pitop/miniscreen/oled/core/contrib/luma/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2014-2019 Richard Hull and contributors
+# See LICENSE.rst for details.
```

Comment:
probably we should include this MIT license

Reply:
a link to where the source code was found may also prove useful, including version/commit hash